model/src/main/java/com/epam/esm/service/impl/GiftCertificateServiceImpl.java

```diff
@@ -63,6 +63,7 @@ public GiftCertificateDto create(GiftCertificateDto giftCertificateDto) {
 	@Override
 	@Transactional
 	public PageDto<GiftCertificateDto> find(Map<String, String> params) {
+		log.debug("Finding GiftCertificate with parameters: {}-------------     -----DEBUG", params);
```

Comment:
what is the point of this debug log if you have same info log below?

Reply:
deleted, this is superfluous